### PR TITLE
431 wording updates

### PR DESCRIPTION
**Pull request by marcellmueller**

### CODE REVIEW

File: app/formSchema/pages/coverage.ts [hunk 1]
```diff
@@ -13,7 +12,7 @@ const mapping = {
     properties: {
       geographicCoverageMap: {
         title:
-          'Use ISED’s Eligibility Mapping Tool to generate the Project Coverage that is to be included in your application.',
+          'Use ISED’s Eligibility Mapping Tool to generate the Project Coverage that is to be included in your application." to "Use ISED’s Eligibility Mapping Tool to generate the project coverage for this application.',
```
Comment (matthieu-foucault) [inline app/formSchema/pages/coverage.ts:15]:
```suggestion
          'Use ISED’s Eligibility Mapping Tool to generate the project coverage for this application.',
```

File: app/components/Review/ReviewField.tsx [hunk 2]
```diff
@@ -1,25 +1,7 @@
 import { FieldProps } from '@rjsf/core';
-import { DateTime } from 'luxon';
 
-const ReviewField: React.FC<FieldProps> = (props) => {
-  const {
-    formContext: { intakeCloseTimestamp },
```
Comment (matthieu-foucault) [inline app/components/Review/ReviewField.tsx:6]: This can now be removed from the `formContext` in `ApplicationForm`

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: app/components/Review/ReviewField.tsx (hunk 2 above)
Comment (marcellmueller) [reply app/components/Review/ReviewField.tsx:6]: Isn't it still being used here?

https://github.com/bcgov/CONN-CCBC-portal/blob/431-wording-updates/app/lib/theme/fields/SubmissionDescriptionField.tsx

Comment (matthieu-foucault) [reply app/components/Review/ReviewField.tsx:6]: good catch

**Review by matthieu-foucault — APPROVED** [review]

**Comment by matthieu-foucault** [comment]
@marcellmueller your commits needs to be signed again

**Pull request merged**